dev-tools/mage/settings.go

```diff
@@ -166,6 +166,10 @@ func initGlobals() {
 
 	ManifestURL = EnvOr(ManifestUrlEnvVar, "")
 	PackagingFromManifest = ManifestURL != ""
+
+	// order matters this must be called last as it will override some of the
+	// values above
+	initPackageVersion()
```

Comment:
We can panic here in case of problems to bail out immediately
```suggestion
	err = initPackageVersion()
	if err != nil {
	  panic(err)
	}
```